workers/unity/Packages/com.improbable.gdk.mobile/Editor/LaunchMenu.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections;
+using System.IO;
+using System.Text;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class LaunchMenu
+    {
+        [MenuItem("SpatialOS/Mobile/Launch on Device", false)]
+        private static void LaunchMobileClient()
+        {
+            EditorUtility.DisplayProgressBar("Launching Mobile Client", "Installing APK", 0.3f);
+
+            // Find apk to install
+            if (!TryGetApkPath("build", out var apkPath))
+            {
+                Debug.LogError("Could not find a build out Android binary to launch.");
+                return;
+            }
+
+            // Install apk on connected phone / emulator
+            RedirectedProcess.Run("adb", "install", "-r", apkPath);
+
+            EditorUtility.DisplayProgressBar("Launching Mobile Client", "Launching Client", 0.9f);
+
+            // Optional arguments to be passed, same as standalone
+            // Use this to pass through the local ip to connect to
+            var runtimeIp = GdkToolsConfiguration.GetOrCreateInstance().RuntimeIp;
+            var arguments = new StringBuilder();
+            if (!string.IsNullOrEmpty(runtimeIp))
+            {
+                arguments.Append($"+receptionistHost {runtimeIp}");
```

Comment:
`+receptionistHost` -> use the default name we store in the RuntimeConfig (think that's what the class is called)